identity/app/views/completeConsents.scala.html

```diff
@@ -1,20 +1,37 @@
+@import com.gu.identity.model.EmailNewsletters
+@import services.EmailPrefsData
+
 @(
     idRequest: services.IdentityRequest,
     idUrlBuilder: services.IdentityUrlBuilder,
-    returnUrl: String
-)(implicit request: RequestHeader)
+    returnUrl: String,
+    emailPrefsForm: Form[EmailPrefsData],
+    emailSubscriptions: List[String],
+    availableLists: EmailNewsletters,
+)(implicit request: RequestHeader, messages: play.api.i18n.Messages)
 
 @import common.LinkTo
 
 <div class="identity-wrapper monocolumn-wrapper">
     <section class="identity-forms-message">
-        <h1 class="identity-title">Thank you for letting us know your preferences</h1>
+        <h1 class="identity-title">Thank you! We've updated your preferences</h1>
         <div class="identity-forms-message__body">
             <p>You can change your preferences anytime by signing in to your account and visiting <a class="u-underline" href="@idUrlBuilder.buildUrl(controllers.editprofile.routes.EditProfileController.displayEmailPrefsForm(false, None).url, idRequest)">Email Preferences</a>.</p>
         </div>
+        <div class="fieldset__fields">
+            <h1 class="identity-title--small">Before you go...</h1>
+            <div>
+                <p>Did you know that we craft <b>over 40</b> different email editions that focus on a vast number of interesting topics - from politics, farming and virtual reality to documentaries, rugby and science breakthroughs?</p>
+                <p>Please take a few seconds to see if there is anything there for you: </p>
+            </div>
+            @fragments.emailListCategories(emailPrefsForm,availableLists,emailSubscriptions)(request, messages)
+            <p class="identity-title-explainer identity-title-explainer--small">
+                The Guardian’s newsletters include content from our website, which may be funded by outside parties. Newsletters may also display information about Guardian News and Media's other products, services or events (such as Guardian Jobs or Masterclasses), chosen charities or online advertisements.
```

Comment:
Minor: there are 2 types of `'` character in this copy.

Reply:
updated